src/components/SellerList.js

```diff
@@ -4,27 +4,18 @@ import styled from 'styled-components';
 import { API } from '../config';
 
 const SellerList = props => {
```

Comment:
- props 에 담겨있는 데이터들은 여기서 바로 구조분해할당해서 사용해주세요!
```suggestion
const SellerList = { sellerList } => {
```

Reply:
옙!